pan3d/ui/toolbar.py

```diff
@@ -22,17 +25,25 @@ def __init__(
                 classes="mx-10",
             )
             with vuetify.VBtn(
-                click=reset,
+                click=reset_function,
                 v_show=unapplied_changes,
-                # small=True,
                 variant="tonal",
             ):
                 html.Span("Apply & Render")
                 html.Span("({{ %s }})" % da_size, v_show=da_size)
-            vuetify.VCheckbox(
-                v_model=(view_edge_visibility, True),
-                v_show=array_active,
-                density="compact",
-                true_icon="mdi-border-all",
-                false_icon="mdi-border-outside",
+            vuetify.VBtn(
+                click="%s = 'Export'" % dialog_shown,
+                variant="tonal",
+                text="Export",
+            )
+            vuetify.VBtn(
+                click="%s = 'Import'" % dialog_shown,
```

Comment:
f-string

Reply:
Changed in [28561cb](https://github.com/Kitware/pan3d/pull/33/commits/28561cb331878bed9f5131df5a01d8307360c41e)